static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -1,64 +1,85 @@
 import FullViewport from 'sentry/components/layouts/fullViewport';
 import * as Layout from 'sentry/components/layouts/thirds';
 import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
 import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import {FeedbackItemLoaderQueryParams} from 'sentry/utils/feedback/item/types';
-import {decodeScalar} from 'sentry/utils/queryString';
-import {useLocation} from 'sentry/utils/useLocation';
+import {decodeList, decodeScalar} from 'sentry/utils/queryString';
+import useLocationQuery from 'sentry/utils/url/useLocationQuery';
 import useOrganization from 'sentry/utils/useOrganization';
 import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
 
 interface Props extends RouteComponentProps<{}, {}, {}> {}
 
 export default function FeedbackListPage({}: Props) {
   const organization = useOrganization();
-  const location = useLocation<FeedbackItemLoaderQueryParams>();
 
-  const queryView = useFeedbackListQueryView({
-    queryReferrer: 'feedback_list_page',
+  const queryView = useLocationQuery({
+    fields: {
+      collapse: ['inbox'],
+      expand: [
+        'owners', // Gives us assignment
+        'stats', // Gives us `firstSeen`
+      ],
+      limit: 25,
+      queryReferrer: 'feedback_list_page',
+      shortIdLookup: 0,
+      end: decodeScalar,
+      environment: decodeList,
+      field: decodeList,
+      project: decodeList,
+      query: decodeScalar,
+      start: decodeScalar,
+      statsPeriod: decodeScalar,
+      utc: decodeScalar,
+    },
+  });
+  const {feedbackSlug} = useLocationQuery({
+    fields: {
+      feedbackSlug: decodeScalar,
+    },
   });
-  const feedbackSlug = decodeScalar(location.query.feedbackSlug);
 
   return (
-    <FeedbackDataContext queryView={queryView} initialDate={new Date()}>
-      <SentryDocumentTitle title={t(`Bug Reports`)} orgSlug={organization.slug}>
-        <FullViewport>
-          <Layout.Header>
-            <Layout.HeaderContent>
-              <Layout.Title>{t('Bug Reports')}</Layout.Title>
-            </Layout.HeaderContent>
-          </Layout.Header>
-          <PageFiltersContainer>
-            <LayoutGrid>
-              <FeedbackFilters style={{gridArea: 'filters'}} />
-              <FeedbackSearch style={{gridArea: 'search'}} />
-              <Container style={{gridArea: 'list'}}>
-                <FeedbackList />
-              </Container>
-              <Container style={{gridArea: 'details'}}>
-                {feedbackSlug ? (
-                  <FeedbackItemLoader feedbackSlug={feedbackSlug} />
-                ) : (
-                  <FeedbackEmptyDetails />
-                )}
-              </Container>
-            </LayoutGrid>
-          </PageFiltersContainer>
-        </FullViewport>
-      </SentryDocumentTitle>
-    </FeedbackDataContext>
```

Comment:
The big change here is that theres a new `<ErrorBoundary>` inside the page, and the FeedbackDataContext is inside of it.

hide whitespace changes to get a better look.